scripts/userstyles.yml

```
@@ -872,6 +872,13 @@ userstyles:
     categories: [discussion_forum, social_networking]
     color: text
     current-maintainers: [*Guaxinim5573]
+  tidal-web:
+    name: Tidal Web
+    link: https://tidal.com
+    categories: [music]
+    icon: tidal
+    color: sapphire
```

```suggestion
    color: text
```

Doesn't seem like they use any color in their branding anymore.